classes/Provider/Gateways/RequestCleaner.php

```diff
@@ -32,7 +32,8 @@ private function clean(array $data)
             if (is_array($value)) {
                 $sanitized[$key] = $this->clean($value);
             } else {
-                $sanitized[$key] = $this->purifier->purify($value);;
+                $sanitized[$key] = $this->purifier->purify($value);
+                ;
```

Comment:
@mdwheele 

There's a fixer for that, too, see https://github.com/FriendsOfPHP/PHP-CS-Fixer!